sessionctx/variable/session.go

```diff
@@ -599,6 +599,9 @@ type SessionVars struct {
 	FoundInPlanCache bool
 	// PrevFoundInPlanCache indicates whether the last statement was found in plan cache.
 	PrevFoundInPlanCache bool
+
+	// OptimizerUseInvisibleIndexes indicates whether optimizer can use invisible index
+	OptimizerUseInvisibleIndexes bool
```

Comment:
Can we put this variable to `StmtCtx`?

Reply:
It seems the only usage of this variable is to make it able to use the invisible index for internal SQLs. Would it impact other internal SQLs like SPM baseline evolution?

Reply:
@zz-jason Now, this flag impact `admin check table` / `admin check index` only.
Any other internal SQLs (like SPM baseline evolution) cannot see invisible index.

Reply:
@eurekaka Comment addressed, PTAL